pkg/ebpftracer/c/headers/common/network.h

```diff
@@ -9,70 +9,79 @@
 
 #include <common/common.h>
 
-// clang-format off
-
 // TYPES
 
 typedef union iphdrs_t {
     struct iphdr iphdr;
     struct ipv6hdr ipv6hdr;
 } iphdrs;
 
-typedef union  {
-    u32 v4addr;
-    unsigned __int128 v6addr;
-}  __attribute__((packed)) addr_t;
+typedef union {
+    __u8 raw[16];
+    __u32 v4addr;
+    __be32 u6_addr32[4];
+} __attribute__((packed)) addr_t;
 
 typedef struct {
     addr_t saddr;
     addr_t daddr;
-    u16 sport;
-    u16 dport;
-    u16 family;
+    __u16 sport;
+    __u16 dport;
+    __u16 family;
 } __attribute__((packed)) tuple_t;
 
-// network flow events
+union addr {
+    __u8 raw[16];
+    __be32 ipv6[4];
+    __be32 ipv4;
+} __attribute__((__packed__));
 
-typedef struct netflow {
-    u32 host_pid;
-    u8 proto;
-    tuple_t tuple;
-} __attribute__((__packed__)) netflow_t;
+typedef struct process_identity {
+    __u32 pid;
+    __u64 pid_start_time;
+    __u64 cgroup_id;
+    __u8 comm[TASK_COMM_LEN];
+} __attribute__((__packed__)) process_identity_t;
 
-statfunc netflow_t invert_netflow(netflow_t flow)
-{
-    tuple_t inverted_tuple = {
-        .saddr = flow.tuple.daddr,
-        .daddr = flow.tuple.saddr,
-        .sport = flow.tuple.dport,
-        .dport = flow.tuple.sport,
-        .family = flow.tuple.family,
-    };
-    netflow_t res = {
-       .host_pid = flow.host_pid,
-       .proto = flow.proto,
-       .tuple = inverted_tuple,
-    };
-    return res;
-}
+struct traffic_summary {
+    size_t rx_packets;
```

Comment:
I'm not a c expert but this size_t was always confusing for me, why not just use `u64` here?

Reply:
`size_t` is normally used to count bytes AFAIK, but I will simply switch to `__u64`. Also if i am not mistaken, the size of `size_t` depends on the underlying arch (x86. vs x64).